update/files/multiFileWriter_test.go

```diff
@@ -33,23 +38,23 @@ func TestNewMultiFileWriter_CheckArgs(t *testing.T) {
 func TestNewFile(t *testing.T) {
 	t.Parallel()
 
-	exportFolderPath := "./"
+	exportFolderPath := "./something"
```

Comment:
This test does not work properly, if, for example, another test uses the same path
Suggestion to change the folder path to a temp dir.

Reply:
this in only inside the tests, should not be problem as it cleans after it.

Reply:
Concurrency problems: imagine 2 tests running in parallel that seldom fails.